Documentation/gettingstarted/k8s-install-openshift-okd.rst

```diff
@@ -118,54 +118,40 @@ Next, generate OpenShift manifests:
 
    openshift-install create manifests --dir "${CLUSTER_NAME}"
 
-Now, define ``cilium`` namespace:
+Next, obtain Cilium manifest from ``cilium/cilium-olm`` repository and copy to ``${CLUSTER_NAME}/manifests``:
 
-.. code:: bash
+.. parsed-literal::
 
-   cat << EOF > "${CLUSTER_NAME}/manifests/cluster-network-03-cilium-namespace.yaml"
-   apiVersion: v1
-   kind: Namespace
-   metadata:
-     name: cilium
-     annotations:
-       # node selector is required to make cilium-operator run on control plane nodes
-       openshift.io/node-selector: ""
-     labels:
-       name: cilium
-       # run level sets priority for Cilium to be deployed prior to other components
-       openshift.io/run-level: "0"
-       # enable cluster logging for Cilium namespace
-       openshift.io/cluster-logging: "true"
-       # enable cluster monitoring for Cilium namespace
-       openshift.io/cluster-monitoring: "true"
-   EOF
+   cilium_olm_rev=master
 
-.. include:: k8s-install-download-release.rst
+   curl --silent --location --fail --show-error "https://github.com/cilium/cilium-olm/archive/${cilium_olm_rev}.tar.gz" --output /tmp/cilium-olm.tgz
+   tar -C /tmp -xf /tmp/cilium-olm.tgz
 
-Next, render Cilium manifest:
+   cp /tmp/cilium-olm-${cilium_olm_rev}/manifests/cilium.v\ |RELEASE|\ /* "${CLUSTER_NAME}/manifests"
 
-.. parsed-literal::
+   rm -rf /tmp/cilium-olm.tgz /tmp/cilium-olm-${cilium_olm_rev}
```

Comment:
Can we be a bit more careful with the `rm -rf something_with_a_${variable}`?
```suggestion
   rm -rf -- /tmp/cilium-olm.tgz "/tmp/cilium-olm-${cilium_olm_rev}"
```

Reply:
Thanks, well spotted! We don't have to have this actually, but I just thought better not to keep too many temp files around....